src/assets/AssetsController.ts

```diff
@@ -668,6 +668,9 @@ export class AssetsController extends BaseController<AssetsConfig, AssetsState>
             return reject(new Error('User rejected to watch the asset.'));
           case SuggestedAssetStatus.failed:
             return reject(new Error(meta.error.message));
+          /* istanbul ignore next */
+          default:
```

Comment:
This is unreachable as far as I can tell, so it couldn't be covered by tests. It should not be a functional change, for the same reason.